src/style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  src: url(fonts/Avenir-Book.ttf);
+}
```

Comment:
that's not how you connect fonts,[ checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 7th item